src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -242,6 +249,11 @@ class PublicationReleasePage extends Component<Props> {
                     id={`content_${order}`}
                     publication={data.publication}
                     refreshCallback={callback => {
+                      const originalOpenAllRefreshCallback = openAllRefreshCallback;
+                      openAllRefreshCallback = () => {
+                        callback();
+                        originalOpenAllRefreshCallback();
```

Comment:
I'm not entirely sure what's going on with this code. The reassignment and `openAllRefreshCallback` being empty function is confusing me particularly.

Reply:
It's being reassigned, as a callback chain each one calling `callback` and the original `openAllRefreshCallback` function. I _could_ put an undefined check in there, but an empty function at the end is cleaner.

Reply:
I don't know how to resolve this as it is a pattern to work this way.